src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -312,6 +320,21 @@ typedef struct {
     int tag_bits;
 } MPIDI_OFI_capabilities_t;
 
+typedef struct {
+    fi_addr_t dest;
+    int rank;
+    int state;
+} MPIDI_OFI_conn_t;
+
+typedef struct MPIDI_OFI_conn_manager_t {
+    int mmaped_size;
```

Comment:
Not to be too picky, but mapped has two p's. This is the past tense of mape, which isn't a word. 😄